microservices/biobuddy/biobuddy/services.py

```diff
@@ -36,9 +39,10 @@ def send_message(request: SendMessageToBioBuddyRequest) -> SendMessageToBioBuddy
     openai_functions = [tool['function'] for tool in request.tools]
 
     strategy_prompt = (f"Given the available tools ({tools_description}), decide whether a direct reply is sufficient "
-                       f"or if a specific plan involving these tools is necessary. If plan is not needed, provide the direct "
-                       f"reply. If plan is needed, reply with the list (with a tag <PLAN> before the list) of function calls in the format <PLAN>: ['Call function_1 in order to do X', 'Call function_2 to achieve Y'].\n\nQuery: \""
-                       f"{request.message_content}\"\n\n")
+                       f"or if a specific plan involving these tools is necessary. If plan is not needed, provide the direct reply. "
```

Comment:
Can we please move prompts and build of prompts to the separate function. Send function already has a lot of responsibilities (call openai, prompts building, decision of what tool to call, etc